tests/tests_integration/test_api/test_time_series.py

```diff
@@ -150,6 +175,70 @@ def test_upsert_2_time_series_one_preexisting(self, cognite_client: CogniteClien
                 external_id=[new_times_series.external_id, preexisting.external_id], ignore_unknown_ids=True
             )
 
+    def test_filter_is_numeric(self, cognite_client: CogniteClient, test_tss: TimeSeriesList) -> None:
+        # Arrange
+        f = filters
+        is_integration_test = f.Prefix(TimeSeriesProperty.external_id, "PYSDK integration test")
+        is_numeric = f.Equals(TimeSeriesProperty.is_string, False)
+
+        # Act
+        result = cognite_client.time_series.filter(
+            f.And(is_integration_test, is_numeric), sort=TimeSeriesProperty.external_id
+        )
+
+        # Assert
+        assert result, "There should be at least one numeric time series"
+
+    def test_aggregate_count(self, cognite_client: CogniteClient, time_series_list: TimeSeriesList) -> None:
+        f = filters
+        is_integration_test = f.Prefix("externalId", "integration_test:")
+
+        count = cognite_client.time_series.aggregate_count(advanced_filter=is_integration_test)
+
+        assert count >= len(time_series_list)
+
+    def test_aggregate_unit(self, cognite_client: CogniteClient, time_series_list: TimeSeriesList) -> None:
+        f = filters
+        is_integration_test = f.Prefix("externalId", "integration_test:")
+
+        count = cognite_client.time_series.aggregate_cardinality_values(TimeSeriesProperty.unit, is_integration_test)
+
+        assert count >= len({t.unit for t in time_series_list if t.unit})
+
+    def test_aggregate_metadata_keys_count(
+        self, cognite_client: CogniteClient, time_series_list: TimeSeriesList
+    ) -> None:
+        f = filters
+        is_integration_test = f.Prefix("externalId", "integration_test:")
+
+        count = cognite_client.time_series.aggregate_cardinality_properties(
+            TimeSeriesProperty.metadata, advanced_filter=is_integration_test
+        )
+
+        assert count >= len({k for t in time_series_list for k in t.metadata.keys()})
+
+    def test_aggregate_unique_units(self, cognite_client: CogniteClient, time_series_list: TimeSeriesList) -> None:
+        f = filters
+        is_integration_test = f.Prefix("externalId", "integration_test:")
+
+        result = cognite_client.time_series.aggregate_unique_values(TimeSeriesProperty.unit, is_integration_test)
+
+        assert set(result.unique) >= {t.unit for t in time_series_list if t.unit}
+
+    def test_aggregate_unique_metadata_keys(
+        self, cognite_client: CogniteClient, time_series_list: TimeSeriesList
+    ) -> None:
+        f = filters
+        is_integration_test = f.Prefix("externalId", "integration_test:")
+
+        result = cognite_client.time_series.aggregate_unique_properties(
+            TimeSeriesProperty.metadata, advanced_filter=is_integration_test
+        )
+
+        assert {tuple(item.value["property"]) for item in result} >= {
+            ("metadata", key.casefold()) for a in time_series_list for key in a.metadata or []
+        }
```

Comment:
When nothing is returned, should this test still pass? Perhaps add an `assert` that it's not empty?
(This question applies to a lot of the tests above as well - and in the similar looking tests in different files)